packages/wrangler/scripts/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This feels a bit brittle. We now have one TS file in this folder that is built with this TS config, but then a load of others that use the top level.
If this script is fundamentally different, then perhaps we should have two folders?
Otherwise I think we should try to keep all these source files compiling with the same config.